### PR TITLE
Improvement/update to webpack 5

### PR DESCRIPTION
**Pull request by stophi-dev**
Update npm dependencies, most notably Webpack to version 5.
We need to test this thoroughly.

Tests:
- [x] Compiling assets work in development
- [x] Compiling assets work in build of software.zip
- [ ] Test all compiler options
- [ ] e2e tests
- [ ] Compare tsc.js of new cplace-asc with tsc.js of old cplace-asc.
- [ ] Source maps: broken ❌ 
- [ ] ...

### CODE REVIEW

**Comment by slaven3kopic** [comment]
@stophi-dev @stefancplace I like the idea 😃  We have it in our backlog as well. Let me know if we can somehow support you

**Comment by stophi-dev** [comment]
@slaven3kopic Sure, you are welcome to review and test this with your use cases.
I already tested a bit and right now I am not aware that this introduces new bugs, but still some tests are pending.

**Comment by slaven3kopic** [comment]
> @slaven3kopic Sure, you are welcome to review and test this with your use cases. I already tested a bit, fixed the obvious bugs and right now I am not aware that this introduces new bugs, but still some tests are pending.

@stophi-dev I will take a look in the next days. @Karthikyadav123 you can maybe start testing this. Please test it on all operating systems (MacOS, Windows, Linux), with different options like `-c`, `-w`. And please test it with different cplace releases as well as with different node versions

**Comment by Karthikyadav123** [comment]
Screen Shot 2022-03-07 at 13 07 39

I am getting an error while compiling with release 5.18. The compilation was unsuccessful. Please find the attached screenshot.

**Comment by stophi-dev** [comment]
@Karthikyadav123 I fixed this bug now.
Unfortunately there is another bug when I run in cplace main the `docker/local/build-image.sh`.
```
Exception in thread "main" org.openapitools.codegen.SpecValidationException: There were issues with the specification. The option can be disabled via validateSpec (Maven/Gradle) or --skip-validate-spec (CLI).
 | Error count: 1, Warning count: 15
Errors:
        -Unable to load RELATIVE ref: schemas/navigation/wm/schemas/navigation/wm/HuiWorkspaceMenu.yaml path: /mnt/c/src/platform/repos/main/cf.cplace.platform/api
Warnings:
        -Unable to load RELATIVE ref: schemas/navigation/wm/schemas/navigation/wm/HuiWorkspaceMenu.yaml path: /mnt/c/src/platform/repos/main/cf.cplace.platform/api

        at org.openapitools.codegen.config.CodegenConfigurator.toContext(CodegenConfigurator.java:556)
        at org.openapitools.codegen.config.CodegenConfigurator.toClientOptInput(CodegenConfigurator.java:583)
        at org.openapitools.codegen.cmd.Generate.execute(Generate.java:433)
        at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
        at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)
✗ Error: [cf.cplace.platform] OpenAPI YAML compilation failed...

✗ Error: [cf.cplace.platform] Failed to write OpenAPI YAML output


✗ COMPILATION FAILED - please check errors in output above


> Task :asc FAILED
```

This seems to be a known bug which block us from updating openapitools.
https://github.com/OpenAPITools/openapi-generator/issues/10320

So I reverted the branch to keep the old version 2.1.26 for `@openapitools/openapi-generator-cli`, but I still get the same error. 🤔

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -1,17 +1,95 @@
...
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout cplace-asc
+        uses: actions/checkout@v3
+
+      - name: Checkout cplace
```
Comment (stefancplace) [inline .github/workflows/continuous-integration.yml:57]: should we also add a test for a product repo e.g. project-planning?

Comment (slaven3kopic) [reply .github/workflows/continuous-integration.yml:57]: Good idea! We should. @stefancplace could you add that?

Comment (stefancplace) [reply .github/workflows/continuous-integration.yml:57]: will do 👍

**Comment by slaven3kopic** [comment]
@stophi-dev This will be implemented in (PR-92)[https://github.com/collaborationFactory/cplace-asc/pull/92]. Thanks for help

**Pull request closed without merging**